Formula/emacs-dracula.rb

```diff
@@ -0,0 +1,20 @@
+class EmacsDracula < Formula
+  desc "Dark color theme available for a number of editors"
+  homepage "https://github.com/dracula/emacs"
+  head "https://github.com/dracula/emacs.git"
+
+  stable do
+    url "https://github.com/dracula/emacs/archive/v1.7.0.tar.gz"
+    sha256 "dbbcc968cf8187a8ada9f040137ba03dc0e51b285e96e128d26cea05cf470330"
+  end
```

Comment:
Considering there isn't any additional stable-only logic to consider. I'd simplify this like this:

```suggestion
  url "https://github.com/dracula/emacs/archive/v1.7.0.tar.gz"
  sha256 "dbbcc968cf8187a8ada9f040137ba03dc0e51b285e96e128d26cea05cf470330"
  head "https://github.com/dracula/emacs.git"
```

It looks like bottles are unnecessary here given there's no compilation involved so I would also add a `bottle :unneeded`:


```suggestion
  url "https://github.com/dracula/emacs/archive/v1.7.0.tar.gz"
  sha256 "dbbcc968cf8187a8ada9f040137ba03dc0e51b285e96e128d26cea05cf470330"
  head "https://github.com/dracula/emacs.git"
  
  bottle :unneeded
```